src/main/kotlin/org/eclipse/tractusx/bpdmcertificatemanagement/dto/request/CertificateDocumentRequestDto.kt

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright (c) 2023 Contributors to the Eclipse Foundation
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Apache License, Version 2.0 which is available at
+ * https://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.eclipse.tractusx.bpdmcertificatemanagement.dto.request
+
+import io.swagger.v3.oas.annotations.media.Schema
+import org.eclipse.tractusx.bpdmcertificatemanagement.dto.CertificateTypeDto
+import org.eclipse.tractusx.bpdmcertificatemanagement.dto.EnclosedSiteDto
+import org.eclipse.tractusx.bpdmcertificatemanagement.dto.TrustValidatorDto
+import java.time.LocalDateTime
+
+@Schema(
+    description = "Certificate document with certificate type",
+    requiredProperties = ["type"]
+)
+data class CertificateDocumentRequestDto(
+
+    val businessPartnerNumber: String? = null,
+    val type: CertificateTypeDto,
+    val registrationNumber: String? = null,
+    val areaOfApplication: String? = null,
+    val remark: String? = null,
+    val enclosedSites: Collection<EnclosedSiteDto>? = emptyList(),
+    val validFrom: LocalDateTime? = null,
```

Comment:
There is an argument to made to use a date type which supports a time zone in order to avoid different time zone conflicts. The user would then need to be explicit about which time zone he actually means when he puts in a date

OffsetDateTime or ZonedDateTime come to mind

Reply:
Thanks for pointing this out :) ZonedDateTime will be perfect choice here and almost forgot these.
So, adapted these with ZonnedDateTime.